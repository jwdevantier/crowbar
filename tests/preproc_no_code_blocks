# <<crowbar
# counter = 0
# @component
# def hello(emit):
#     global counter
#     counter += 1
#     emit(f"hello{counter}")
# >>
# <<end>>


indent 0, line
# <<crowbar
# emit(hello())
# >>
# <<end>>

indent 0, multiline
/* <<crowbar
   emit(hello())
   >> */
/* <<end>> */

indent 3
   # <<crowbar
   # emit(hello())
   # >>
   # <<end>>

indent 3, multiline
   /* <<crowbar
      emit(hello())
      >> */
   /* <<end>> */
