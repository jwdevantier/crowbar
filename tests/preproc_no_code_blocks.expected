

indent 0, line
hello1

indent 0, multiline
hello2

indent 3
   hello3

indent 3, multiline
   hello4
